package.json

```diff
@@ -9,15 +12,9 @@
     "@guardian/eslint-config": "file:packages/eslint-config",
     "@guardian/eslint-config-typescript": "file:packages/eslint-config-typescript",
     "@guardian/prettier": "file:packages/prettier",
-    "@typescript-eslint/eslint-plugin": "^4.26.0",
-    "@typescript-eslint/parser": "^4.26.0",
-    "eslint": "^7.28.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-eslint-comments": "^3.2.0",
-    "eslint-plugin-import": "^2.23.4",
-    "eslint-plugin-prettier": "^3.4.0",
+    "eslint": "8.0.0",
```

Comment:
[8.4.1](https://www.npmjs.com/package/eslint) is the latest, should we be using that instead?

Reply:
The idea was to install the minimum possible version in the workspace that our packages should work with, to make using them as flexible as possible without accidentally using features added after the minimum version